src/uu/cp/Cargo.toml

```diff
@@ -28,7 +28,7 @@ uucore = { version=">=0.0.11", package="uucore", path="../../uucore", features=[
 walkdir = "2.2"
 
 [target.'cfg(target_os = "linux")'.dependencies]
-ioctl-sys = "0.6"
+ioctl-sys = { git = "https://github.com/jmesmon/ioctl.git", rev = "295ed33a8704a0eff31bebdcf6ca0bfeb93bf675" }
```

Comment:
I am uncomfortable depending on some personal forked git repo.
So, I won't accept this change. Sorry :( 
Do you know why it isn't merged upstream + published as a new crate?
thanks

Reply:
I'm not sure why a new release hasn't been made yet, but that repo is actually the upstream, official ioctl-sys. 
I'll ask them about updating the crate.

Reply:
OK, thanks for https://github.com/jmesmon/ioctl/issues/22 :)

Reply:
A new crate release was just made and [jmesmon/ioctl#22](https://github.com/jmesmon/ioctl/issues/22) was resolved, so I've amended commit 4345c2a821cfd4ecceeab3158dec62d4cd1f8a3f to use the new version 0.8.0.